src/test_meta_srv.c

```diff
@@ -90,7 +90,7 @@ int test_meta_srv(Connection *ctl, tcp_stat_agent *agent,
                          testOptions->connection_flags);
       if (err) {
         // message reading error
-        log_println(0, "Protocol error!");
+        log_println(0, "Protocol error! (%d)", err);
```

Comment:
Nit: err is int64_t (mismatched with format), and it looks like it only needs to be an int. Suggest changing the variable type.